python/lsst/analysis/tools/actions/plot/focalPlanePlot.py

```diff
@@ -168,8 +183,14 @@ def makePlot(
             noDataFig = addPlotInfo(noDataFig, plotInfo)
             return noDataFig
 
-        fig = plt.figure(dpi=300)
-        ax = fig.add_subplot(111)
+        if self.addHistogram:
+            fig = plt.figure(dpi=300, figsize=(12, 6))
+            gs = gridspec.GridSpec(1, 2, width_ratios=[3, 2])
+            ax = fig.add_subplot(gs[0, 0])
+            histAx = fig.add_subplot(gs[0, 1])
+        else:
+            fig = plt.figure(dpi=300)
```

Comment:
We should probably make dpi and figsize configurable some day.

Reply:
Agreed